### ISSUE TITLE
DKIM configuration seems somehow broken

### ISSUE THREAD

**Issue by kelson42** [kiwix/k8s#109]
If I trust https://mxtoolbox.com/problem/dkim/dkim-record-published?page=prob_dkim&action=dkim:kiwix.org:email&showlogin=1&hidepitch=0&hidetoc=1

**Comment by benoit74** [comment]
From what I see, no DKIM record is set up at all.

We have to do it for all our service providers, list to be confirmed first (see #87). A quick look seems to indicate that Gaggle has no idea about how to do it.

**Comment by rgaudin** [comment]
> From what I see, no DKIM record is set up at all.

That's incorrect. We have DKIM records for Gandi SMTP, Freshdesk, all of Mailgun's subdomains and a bunch of stuff that are probably obsolete.

It's possible some obsolete stuff are polluting the zone as we are allowed up to 3 DKIM per domain. kiwix zone is a mess

As DKIM are per-sending-service public key, keys are to be checked individually.

**DKIM tester**: https://dmarcly.com/tools/dkim-record-checker

- `cardshop.hotspot.kiwix.org` / `smtp` (Cardshop Mailgun)
- `wp.kiwix.org` / `s1` (Wordpress Mailgun)
- `kiwix.org` / `gm1`|`gm2`|`gm3` (Gandi SMTP)
